src/main/java/com/shop/console/ConsoleApp.java

```diff
@@ -0,0 +1,80 @@
+package com.shop.console;
+
+import com.shop.ShopStorageMemory;
+import com.shop.console.utils.Commands;
+import com.shop.console.utils.Menu;
+import com.shop.dto.Catalog;
+import com.shop.dto.Category;
+
+import static com.shop.console.utils.Input.*;
+
+public class ConsoleApp {
```

Comment:
Main class shouldn't contain any methods expect main method. You need to move this logic in service class (try to think which one).